# the robot position 
float32 x
float32 y

# the velocity of the robot
float32 vel_x
float32 vel_z